reference-stores/next/pages/collections/[handle].js

```diff
@@ -50,11 +50,11 @@ export async function getStaticProps({ params: { handle } }) {
   });
   const { page } = await resolvePageData({
     client: nacelleClient,
-    page: pages.edges[0]?.node
+    page: data.pages.edges[0]?.node
   });
   return {
     props: {
-      collection: collections.edges[0].node || null,
+      collection: data.collections.edges[0].node || null,
```

Comment:
Might be a good opportunity to use optional chaining with `.at`:

```suggestion
      collection: data.collections.edges.at(0)?.node || null,
```